File: SPCode/UI/Support/SPRadioButtonListEditorUI.vb
Imports System.ComponentModel
Imports System.ComponentModel.Design
Imports System.Drawing.Design
Imports System.Windows.Forms
Imports System.Windows.Forms.Design
Imports System.Windows.Forms.ComponentModel

Namespace UI
    Public Class SPRadioButtonListEditorUI

        Private _RadioItems As SPRadioButtonItemCollection = Nothing

        Friend Sub New(ByVal itemsTarget As SPRadioButtonItemCollection, _
        ByVal editorService As IWindowsFormsEditorService)

            InitializeComponent()

        End Sub

        ' LightShape is the property for which this control provides
        ' a custom user interface in the Properties window.
        Public Property RadioItems() As SPRadioButtonItemCollection

            Get
                Return Me._RadioItems
            End Get

            Set(ByVal Value As SPRadioButtonItemCollection)
                If Me._RadioItems IsNot Value Then
                    Me._RadioItems = Value
                End If
            End Set
        End Property
    End Class
End Namespace

File: SPCode/CS/UI/Support/SPRadioButtonListEditorUI.cs
using System.Text.RegularExpressions;
using System.Diagnostics;
using System;
using System.Drawing.Drawing2D;
using System.Windows.Forms;
using System.Collections;
using System.Drawing;
using Microsoft.VisualBasic;
using System.Data.SqlClient;
using System.Data;
using System.Collections.Generic;
using WeifenLuo.WinFormsUI;
using Microsoft.Win32;
using WeifenLuo;
using System.ComponentModel;
using System.ComponentModel.Design;
using System.Drawing.Design;
using System.Windows.Forms.Design;
using System.Windows.Forms.ComponentModel;


namespace SoftLogik.Win
{
	namespace UI
	{
		public partial class SPRadioButtonListEditorUI
		{
			
			private SPRadioButtonItemCollection _RadioItems = null;
			
			internal SPRadioButtonListEditorUI(SPRadioButtonItemCollection itemsTarget, IWindowsFormsEditorService editorService)
			{
				
				InitializeComponent();
				
			}
			
			// LightShape is the property for which this control provides
			// a custom user interface in the Properties window.
			public SPRadioButtonItemCollection RadioItems
			{
				
				get
				{
					return this._RadioItems;
				}
				
				set
				{
					if (!(this._RadioItems is value))
					{
						this._RadioItems = value;
					}
				}
			}
		}
	}
	
	
	
}
